
			// Print the message from MyException object